inc/class-ses.php

```diff
@@ -20,20 +21,22 @@ public static function get_instance() {
 
 		if ( ! self::$instance ) {
 
-			$key    = defined( 'AWS_SES_WP_MAIL_KEY' ) ? AWS_SES_WP_MAIL_KEY : null;
-			$secret = defined( 'AWS_SES_WP_MAIL_SECRET' ) ? AWS_SES_WP_MAIL_SECRET : null;
-			$region = defined( 'AWS_SES_WP_MAIL_REGION' ) ? AWS_SES_WP_MAIL_REGION : null;
+			$key    	= defined( 'AWS_SES_WP_MAIL_KEY' ) ? AWS_SES_WP_MAIL_KEY : null;
+			$secret 	= defined( 'AWS_SES_WP_MAIL_SECRET' ) ? AWS_SES_WP_MAIL_SECRET : null;
+			$region 	= defined( 'AWS_SES_WP_MAIL_REGION' ) ? AWS_SES_WP_MAIL_REGION : null;
+			$config_set = defined( 'AWS_SES_WP_MAIL_CONFIG_SET' ) ? AWS_SES_WP_MAIL_CONFIG_SET : null;
 
 			self::$instance = new static( $key, $secret, $region );
 		}
 
 		return self::$instance;
 	}
 
-	public function __construct( $key, $secret, $region = null ) {
-		$this->key = $key;
-		$this->secret = $secret;
-		$this->region = $region;
+	public function __construct( $key, $secret, $region = null, $config_set = "" ) {
```

Comment:
The fallback value for `$region` is `null`, and the fallback for the config set constant is `null` so lets keep this consistent.
```suggestion
	public function __construct( $key, $secret, $region = null, $config_set = null ) {
```